### DISCUSSION TITLE
How to apply redux-micro-frontend with nextjs

### DISCUSSION THREAD

**Discussion by quynhbui-teqnological** [microsoft/redux-micro-frontend#28] [General]
Your library is very well but i don't know how to apply it in my project
I am building a project by micro-front-end with nextjs. But in progress, we see an issue about managing redux of each shared component: `Error: could not find react-redux context value; please ensure the component is wrapped in a <Provider>`. I think , they can't able to see the state of other apps, need to set up to share state between components

Can you guide me how to apply your library in micro-front-end with nextjs?

**Comment by patrickCode** [comment]
This error generally happens if you are trying to use react-redux to create higher order components. This library doesn't is dependent only of `redux` and not `react-redux` so there shouldn't be any issue in using it.

However, if you are using `react-redux` for creating higher order components then you will need to wrap the parent component with a `<Provider>` tag.
